backend/btrixcloud/models.py

```diff
@@ -259,6 +259,7 @@ class UpdateCrawlConfig(BaseModel):
     tags: Optional[List[str]] = None
     description: Optional[str] = None
     autoAddCollections: Optional[List[UUID4]] = None
+    runNow: Optional[bool] = False
```

Comment:
`Optional` in Python typing means "can be None", so in the case of booleans maybe we just want to make this `runNow: bool = False`. Since we're setting a default for it, if it's unset it'll be False.

Reply:
Ohh, good call!